### PR TITLE
docs: Detail what systems need to be added to the new AKS installation

### PR DESCRIPTION
**Pull request by draychev**
With this small tweak we outline what mutations we are going to do to our new AKS cluster.
Gives a quick overview at a glance of the required Kubernetes changes.

### CODE REVIEW

File: docs/install-new.md [hunk 1]
```diff
@@ -46,9 +46,13 @@ The steps below require the following software to be installed on your workstati
 
 ## Setting up Application Gateway Ingress Controller on AKS
 
-Once, we have the created the resources in Azure, we need to deploy following controllers on the cluster:
-1) `aad pod identity` - This controller uses user assigned identity and provide ARM access token to the controller.
-2) `application gateway ingress controller` - This controller communicates ingress related events to the Application Gateway resource.
+### Overview
+With the instructions in the previous section we created and configured a new Azure Kubernetes Service (AKS) cluster. We are now ready to deploy new Kubernetes infrastructure. The instructions below will guide us through the proccess of installing the following 2 components on our new AKS:
+1. **Azure Active Directory Pod Identity** - Provides token-based access to the [Azure Resource Manager (ARM)](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview) via user-assigned identity. Adding this system will result in the installation of the following within your AKS cluster:
+   1. custom Kubernetes resource definitions: `AzureIdentity`, `AzureAssignedIdentity`, `AzureIdentityBinding`
```
Comment (akshaysngupta) [inline docs/install-new.md]: *Custom

File: docs/install-new.md [hunk 2]
```diff
@@ -46,9 +46,13 @@ The steps below require the following software to be installed on your workstati
 
 ## Setting up Application Gateway Ingress Controller on AKS
 
-Once, we have the created the resources in Azure, we need to deploy following controllers on the cluster:
-1) `aad pod identity` - This controller uses user assigned identity and provide ARM access token to the controller.
-2) `application gateway ingress controller` - This controller communicates ingress related events to the Application Gateway resource.
+### Overview
+With the instructions in the previous section we created and configured a new Azure Kubernetes Service (AKS) cluster. We are now ready to deploy new Kubernetes infrastructure. The instructions below will guide us through the proccess of installing the following 2 components on our new AKS:
```
Comment (akshaysngupta) [inline docs/install-new.md]: is it suppose to be 'ready to deploy `to our` new Kubernetes infrastructure' ?

Comment (draychev) [reply docs/install-new.md]: I like this clarification - thanks!

**Comment by draychev** [comment]
@akshaysngupta PTAL

**Pull request merged**